ob-yaml.md

```diff
@@ -139,6 +139,8 @@ orderbooks:
 
 At the minimum a token is a network, address and decimals. While ERC20 doesn't mandate decimals on the contract, it is required in practise to allow for standardized decimal 18 math, as non-18 decimal token values need to be scaled up/down to match. If a token has `0` decimals then this implies there are no fractional values and any amounts are to be treated as literal integers.
 
+> **Note:** The `tokens` mapping is optional and can be omitted if no tokens need to be predefined. If tokens are selected via the GUI, this mapping will be generated automatically upon selection.
```

Comment:
I think we can be clearer here

This top level field may be omitted as orders may reference tokens from other sources, e.g.:
- `using-tokens-from`
- those provided by an application use `select-token` as part of the top-level `gui` field.